scalameta/dialects/shared/src/main/scala/scala/meta/Dialect.scala

```diff
@@ -68,7 +68,9 @@ final class Dialect private (
     // Normally none is required, but scripts may have their own rules.
     val toplevelSeparator: String,
     // Are numeric literal underscore separators, i.e. `1_000_000` legal or not?
-    val allowNumericLiteralUnderscoreSeparators: Boolean
+    val allowNumericLiteralUnderscoreSeparators: Boolean,
+    // Literal Unit Type
+    val allowLiteralUnitType: Boolean
```

Comment:
is this something that works for 2.12 and 2.13, but not for 2.11 ?

Reply:
hah it only works for 2.13 I realized that later but forgot to change PR title.
Yes it is something that only works for 2.13